pac4j-saml/src/main/java/org/pac4j/saml/config/SAML2Configuration.java

```diff
@@ -860,6 +902,13 @@ public SAML2MetadataGenerator toMetadataGenerator() {
         }
     }
 
+    protected void determineSingleSignOutServiceUrl(final BaseSAML2MetadataGenerator generator) {
+        final String url = StringUtils.defaultIfBlank(this.singleSignOutServiceUrl, callbackUrl);
+        final String logoutUrl = CommonHelper.addParameter(url, Pac4jConstants.LOGOUT_ENDPOINT_PARAMETER, "true");
```

Comment:
What could be the use case here? Why would you receive SAML logout requests on another endpoint than the callback URL which knows how to handle that?

Reply:
The comment is outdated also.

Reply:
First, thanks for the spring cleaning!

Often times, yes the endpoint that does the callback is responsible for handling both authn responses and logout responses. There are scenarios where you do not want them to be the same: idp posts an invalid logout response back, or one would want to handle the logout response rather differently via a dedicated url (that redirects to places, etc). So this allows one to separate the normal authn response handling and also potentially removes the need foe logoutendpoint parameter (which can become problematic as certain idps remove parameters when they post a logout response, which then causes destination values to not match)

Reply:
OK